steelthread-0/walkthrough.md

```diff
@@ -15,123 +18,234 @@ The installation has already been performed on an OpenShift cluster as per the i
+This is a webapp that connects to the EnMasse address space. It allows the user to add new fruit to the inventory of the CRUD app via the EnMasse and CRUD app Integration in Fuse.
 
-```
+Load up the launcher application. This is at ```https://launcher-launcher.<ROUTER_SUFFIX> ``` 
 
-set up the templates in the cluster
+- Click on launch your project
+- Choose ```eval``` as the application name again
+- When prompted to select target environment choose : ```Code Locally, Build and Deploy```
+- Click the blue down arrows
+- Under mission select ```Messaging Work Queue```
+- Under runtime select ```Node.js```
+- Choose the ```integreatly``` option in the dropdown
+- Click the down arrows
+- Under the repository input change the value to ```messaging-frontend```
+- Wait for the down arrows to change color to blue, then click them
+- Click setup application.
+- This will create a repo called messaging-frontend in your github account and deploy the Messaging booster to the eval namespace.
 
-```
-cd nodejs-messaging-work-queue
-oc apply -f templates
+There a some credentials to set to allow the messaging frontend to connect to the EnMasse address space.
+First, get the `username` and `password` values from the EnMasse binding secret.
+It has a name in the format `enmasse-standard-xxxxx-credentials-yyyyy`
 
-```
-Deploy the applications
+Edit the `messaging-service` configmap, setting the `MESSAGING_SERVICE_PASSWORD` & `MESSAGING_SERVICE_USER` to the values from the EnMasse binding secret.
 
 ```
-oc new-app --template=nodejs-messaging-work-queue-frontend
-oc new-app --template=nodejs-messaging-work-queue-worker
+oc edit cm messaging-service -n eval
 ```
 
-Edit the messaging-service configmap
+Restart the booster services
 
 ```
-oc edit cm messaging-service -n eval:
+oc rollout latest nodejs-messaging-work-queue-frontend -n eval
 ```
 
-Add in the values from the secret from the Enmasse binding secret
-
-Set the following key value pairs
+In the log of the application you should see something like:
 
 ```
-MESSAGING_SERVICE_HOST = <value from messagingHost>
-MESSAGING_SERVICE_PASSWORD = <value from password>
-MESSAGING_SERVICE_USER= <value from username>
+frontend-nodejs-aad7: Connected to AMQP messaging service at messaging.enmasse-eval.svc:5672
 ```
 
-Restart the booster services
+# Setup Enmasse connection in Fuse Ignite
 
-```
-oc rollout latest nodejs-messaging-work-queue-frontend
-oc rollout latest nodejs-messaging-work-queue-worker
-```
+You will need the `username` and `password` credentials again for this section from the EnMasse binding secret (`enmasse-standard-xxxxx-credentials-yyyyy`)
+
+- Open the Fuse Ignite console. You will find the route in the OpenShift Web Console overview screen for the `eval` project.
+- Login in to the Fuse Ignite console using the evals@example.com user 
+- Select `Connections` and `Create connection`
+- Find the `AMQP Message Broker` connector and select it. 
```

Comment:
I made the mistake of selecting ```AMQ Message Broker```. Might be worth adding a note saying to ensure it's ```AMQP``` and not ```AMQ```. Chances are others will do the same